web/src/main/resources/org/akaza/openclinica/i18n/words.properties

```diff
@@ -3094,7 +3096,3 @@ preview=Preview
 choose_crf_migration_batch_instruction_key=Please choose the current CRF Version, New CRF Version, applicable Sites, and applicable Events for batch CRF version migration. This action will be audited and appear in each Subject's audit log.
 migrate=Migrate
 
-repeating_layout_group_item_assigned_to_wrong_group=All items located directly or indirectly in a repeating layout group must be assigned to the same data group.
-nested_repeat_group_not_allowed=Repeating layout groups can be included at most one time in a nested layout groups structure
-duplicate_item_name=Item Names are unique per CRF
-group_name_missing_for_this_item=CRFs must have a data group defined for every Item 
```

Comment:
This group of strings was defined in a 4.x commit and will not exist when the pull request is re-created against 3.13